hibernate-java8/src/main/java/org/hibernate/type/OffsetDateTimeType.java

```diff
@@ -53,7 +54,7 @@ public OffsetDateTime next(OffsetDateTime current, SessionImplementor session) {
 	@Override
 	@SuppressWarnings("unchecked")
 	public Comparator<OffsetDateTime> getComparator() {
-		return ComparableComparator.INSTANCE;
+		return OffsetDateTimeComparator.INSTANCE;
```

Comment:
We could just use java.time.OffsetDateTime#timeLineOrder which is a `Comparator<OffsetDateTime>`